src/app/COMPONENTS/login-page/login-page.component.ts

```diff
@@ -24,9 +26,12 @@ export class LoginPageComponent {
 
   isServer: boolean | null = null
 
-  constructor(private fb: FormBuilder, private userService: UsersService, private router: Router, @Inject(PLATFORM_ID) platformId: Object) {
+  constructor(private fb: FormBuilder, private userService: UsersService, private router: Router, @Inject(PLATFORM_ID) platformId: Object, public translate: TranslateService) {
     this.isServer = isPlatformServer(platformId)
 
+    this.translate.addLangs(['en', 'ru', 'es', 'lv']);
```

Comment:
you should move all the language functionality to the app component, better yet you can create an app.service.ts for this logic, since you would need to get the language when you open any page of the app